create-resources-at-scale/create-resources-at-scale.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Passed validation in Azure Cloud Shell Bash environment on 4/25/2022
+
+# Variable block
+# These variables have placeholder values that are replaced with values from the csv input file.
+Identifier="0"
+user=""
+createvm="true"
+createvnet="true"
+createrg="true"
+subnet=""
+vmName="msdocs-linuxVM-$user"
+vnetname=""
+vmImage="Ubuntu2204"
+location=""
+resourceGroup="msdocs-ubuntu-vm-group22-$Identifier"
+
+# task 1 create resources
+for i in $(seq 0 1);
+do
+    # Read in commands from csv file
+    echo read in commands from csv file
+    c1= sed -n "$Identifier"p file.csv

```

Comment:
When reading from a CSV or JSON file, don't you need to use [jq](https://jqlang.github.io/jq/manual/)?

# Show the contents of a file in the console
fileName=msdocs-tutorial.json
cat $fileName | jq

# Get a JSON dictionary object
stgKV=$(jq -r '.environments.stg."kv-secretName"' $fileName)
echo $stgKV

# Filter a JSON array
devKV=$(jq -r '.environments.dev[] | select(.status=="active") | ."kv-secretName"' $fileName)
echo $devKV